dynamicgrid_test.go

```diff
@@ -96,7 +96,7 @@ func TestDynamicGrid(t *testing.T) {
 	}
 	results := r["TotalPopD"]
 	totald := floats.Sum(results)
-	const expectedDeaths = 1.706171742850251e-05
+	const expectedDeaths = 17.061717399701948
```

Comment:
I think the expected deaths was super small (1.7 x 10 ^-5) before because the mortality rate was really small. It was being decreased below the minimum mortality rate in the test grid through the area-weighted averaging. Does that make sense?